If Qt HiDpi scaling is active, use that insted of GRID_UNIT_PX

mariogrip opened the pull request:
This is the first move to depricate GRID_UNIT_PX all togheter

mariogrip committed:
If Qt HiDpi scaling is active, use that insted of GRID_UNIT_PX

This is the first move to depricate GRID_UNIT_PX all togheter

mardy commented on src/UbuntuToolkit/ucunits.cpp:
```diff
@@ -322,8 +326,16 @@ void UCUnits::windowPropertyChanged(QPlatformWindow *window, const QString &prop
+    if (QHighDpiScaling::isActive())
+      setGridUnit(qCeil(m_devicePixelRatio * DEFAULT_GRID_UNIT_PX));
+    else
+      setGridUnit(qCeil(scale * DEFAULT_GRID_UNIT_PX));
```
Shouldn't we keep this line as it was before, and multiply the value by `m_devicePixelRatio`?

mariogrip replied:
I though so too, but it seems to always return 1 if HighDpiScaling is disabled. I might be wrong here, but that's what i got from reading the qt's src.

mariogrip replied:
Looking at http://code.qt.io/cgit/qt/qtbase.git/tree/src/gui/kernel/qhighdpiscaling.cpp?h=5.9.6#n399 it seems to return 1

mardy replied:
Just to play 100% safe, I'd add it back, but it's your call :-)

Edit: OK, then, keep it as it is :-)

mariogrip committed:
Add back m_devicePixelRatio to make sure we dont make regression

devicePixelRatio should always be 1 if qts hidpi is disabled afaik, but lets add it back to be sure
